core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -969,4 +992,7 @@ private[spark] object Executor {
   // task is fully deserialized. When possible, the TaskContext.getLocalProperty call should be
   // used instead.
   val taskDeserializationProps: ThreadLocal[Properties] = new ThreadLocal[Properties]
+
+  val MDC_KEY = "mdc."
+  val TASK_MDC_KEY = s"${MDC_KEY}taskName"
```

Comment:
if you change this key you need also to update the docs

Reply:
IIUC, the change doesn't introduce any difference to the end-user, no?

Reply:
as I remember before it was just taskName without mdc.taskName

Reply:
This is used internally in order to handle taskName consistently with custom MDC properties. It has no effect to users.

Reply:
It affects end-users, as now they need to use `mdc.taskName` in the log4j config file, instead of `taskName`.

But it's not released yet, I'm fine to make it consistent that all the MDC properties start with `mdc.`

Reply:
ok